x/gov/simulation/operations_test.go

```diff
@@ -131,7 +131,7 @@ func TestSimulateMsgSubmitProposal(t *testing.T) {
 }
 
 // TestSimulateMsgDeposit tests the normal scenario of a valid message of type TypeMsgDeposit.
-// Abnormal scenarios, where the message is created by an errors are not tested here.
+// Abnormal scenarios, where the message is created by the errors are not tested here.
```

Comment:
ditto for all the similar comments below
```suggestion
// Abnormal scenarios, where errors occur, are not tested here.
```